MAUI/Calendar/migration.md

```diff
@@ -884,8 +884,8 @@ this.Content = calendar;
 </tr>
 <tr>
 <td>{{'[LayoutBackground](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.YearViewSettings.html#Syncfusion_SfCalendar_XForms_YearViewSettings_LayoutBackground)'| markdownify }}</td>
-<td>Nil</td>
-<td>Not Supported.</td>
+<td>{{'[Background](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarMonthView.html#Syncfusion_Maui_Calendar_CalendarMonthView_Background)'| markdownify }}</td>
+<td>Gets or sets the background for the month cells of the calendar month view.</td>
 </tr>
```

Comment:
NavigateToMonthOnInActiveDatesSelection | Nil | Not Supported.
-- | -- | --


[NavigateToMonthOnInActiveDatesSelection](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SfCalendar.html#Syncfusion_SfCalendar_XForms_SfCalendar_NavigateToMonthOnInActiveDatesSelection)	Nil	Not Supported.

Can achive using Ontap and Display date API?